### ISSUE TITLE
Implement `.pop()` method

### ISSUE THREAD

**Issue by adamn** [js-cookie/js-cookie#294]
It would be nice to have a `pop()` method which would essentially run `get()` and `remove()` in a robust way.

**Comment by FagnerMartinsBrack** [comment]
isn't `pop` a terminology used just to remove the head of a queue mechanism? In this case `Cookies` represents a data storage mechanism so it might not make sense having a `.pop()` method, does it?

Besides, one is free to wrap the `Cookies` in a domain model so that it's possible to consume an internal abstraction throughout the system without having to use the `Cookies` APIs directly. That can also be considered a good practice if you want custom methods:

```javascript
const MyCustomCookieStorage = () => {
  pop: () => {
    Cookies.get();
    Cookies.remove();
  }
}
```

People have requested to add other methods to the `Cookies` namespace before, see https://github.com/js-cookie/js-cookie/pull/123. In that case, we ended up deciding to create a separate project because that use case didn't seem too relevant for the majority of js-cookie consumers.

Based on that feedback and this one, what we can do is to build a plugin API that makes it easy to create a new functionality in js-cookie. Right now it's not as trivial to maintain the library correctness while extending it unless you wrap in your domain model as I mentioned in the previous example. Last time I tried something like this was in the same `js-cookie-remove-all` project, see https://github.com/luanpotter/js-cookie-remove-all/pull/1/files.

I just don't think it's worth doing that given the cost of new code we need to support and the amount of bytes it might add everyone using the library now. If you want to go that path, feel free to create a PR.
